lib/Dialect/FIRRTL/Transforms/CreateSiFiveMetadata.cpp

```diff
@@ -41,6 +41,8 @@ class CreateSiFiveMetadataPass
   DenseSet<Operation *> dutModuleSet;
   // The design under test module.
   FModuleOp dutMod;
+  CircuitOp circuitOp;
+  mlir::ModuleOp moduleOp;
```

Comment:
I don't think anything uses this, can it just be a local variable where it is needed?